terraform/rds/bind/postgresql/provider.tf

```diff
@@ -19,5 +19,5 @@ provider "postgresql" {
   password  = var.admin_password
   superuser = false
   database  = var.db_name
-  sslmode   = var.use_tls ? "require" : "disable"
+  sslmode   = "verify-ca"
```

Comment:
Are we thinking the cost of installing the AWS CA bundle is so low that even in dev situations this would be acceptable, rather than still allowing SSL to be disabled?

Reply:
That's the idea, but I'm happy for that to be challenged. It's not hard to retain this switch.

Reply:
If we did retain the switch it would be:
```terraform
sslmode = var.use_tls ? "verify-ca" : "allow"
```

Reply:
Maybe changing to `var.use_tls ? "verify-ca" : "disable"` increases the security, but still allows for a dev case where potentially the user doesn't have the ability to easily install the CA bundle.
I'm not speaking from experience of encountering this issue, so maybe this is a useless switch, I just feel given it was previously an option, we could retain that functionality.

Reply:
Sorry @blgm - I didn't refresh the page and see your comment before making the same suggestion!

Reply:
The reasoning is that "allow" will use a TLS connection if it can, but will drop down to a non-TLS connection if it can't. Whereas "disable" will try an non-TLS connection and will fail if it can't get one. Given that user passwords are being sent over the connection, I'd rather it defaulted to at least trying TLS.